src/containers/pages/IRS/Map/index.tsx

```diff
@@ -298,7 +298,13 @@ const IRSReportingMap = (props: IRSReportingMapProps & RouteComponentProps<Route
                     value={row.value}
                   />
                   <p className="indicator-breakdown">
-                    {PROGRESS}: {row.numerator} {OF} {row.denominator} {`${row.unit || STRUCTURE}s`}{' '}
+                    {PROGRESS}:{' '}
+                    {format(
+                      NUMERATOR_OF_DENOMINATOR_UNITS,
+                      row.numerator,
+                      row.denominator,
+                      row.unit || STRUCTURES
+                    )}{' '}
                     ({row.value}%)
```

Comment:
In this case, it felt like compound strings made sense given the nature of the information (`<metadata>:<textual data> (<numerical data>)`) and are translated separately (or not at all).